west.yml

```diff
@@ -111,7 +111,7 @@ manifest:
       revision: 3f545d76a2e6d1db83a470ccdb5bebd1f484e137
       path: modules/lib/loramac-node
     - name: openthread
-      revision: 7854c687bf23ce509a65d118cbea0c7011a89a1e
+      revision:pull/31/head
```

Comment:
Space is missing after `revision:`, causing the CI failure.

Reply:
fixed.